lsp-test/src/Language/LSP/Test.hs

```diff
@@ -241,7 +241,7 @@ runSessionWithHandles' serverProc serverIn serverOut config' caps rootDir sessio
   exitServer :: Session ()
   exitServer = request_ SShutdown Empty >> sendNotification SExit Empty
 
-  -- | Listens to the server output until the shutdown ack,
+  -- | Listens to the server output until the shutdown hack,
```

Comment:
This is wrong, it was "ack" as in "acknowledges"

Reply:
could it be typed as `SHUTDOWN ACK`

Reply:
I think it's clear enough to me, but I wouldn't object to ACK.

Reply:
You can just push more commits to your branch, and it will update the PR.

Reply:
don't you have to squash

Reply:
No.